definitions/checks/foreman_login_session.rb

```diff
@@ -0,0 +1,21 @@
+class Checks::ForemanLoginSession < ForemanMaintain::Check
+  metadata do
+    label :foreman_login_session
+    tags :foreman_login_session
+    description 'Check the login session count'
+  end
+
+  def run
+    login_session_count = count
+    assert(login_session_count <= 2_151_782_769,
```

Comment:
@patilsuraj767 Is the value the max id that we can't pass? Would it make sense to set the value to something smaller to increase the chance we catch the situation before it is too late e.g. during the upgrade or regular health checks? Can we count how quickly it grows and estimate when it is going to fail?

Also could we make the value named constant (e.g. MAX_SESSION_ID) to see what we actually match against?

Reply:
Yes, I will make it as a constant (MAX_SESSION_ID).
Max value we can't pass is 2151782969. Should we set it to 1075891484 (2151782969/2)?